src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -89,3 +90,63 @@
                                                 {:event  :wallet/get-suggested-routes
                                                  :error  error
                                                  :params request-params}))}]})))
+
+(rf/reg-event-fx :wallet/add-authorized-transaction
+ (fn [{:keys [db]} [transaction]]
+   (let [transaction-hashes (:hashes transaction)
+         chain-id           (key (first transaction-hashes))
+         tx-id              (first (val (first transaction-hashes)))
+         transaction-detes  {:status   :pending
+                             :id       (:id transaction)
+                             :chain-id chain-id}]
+     {:db (assoc-in db [:wallet :transactions tx-id] transaction-detes)
+      :fx [[:dispatch [:pop-to-root :shell-stack]]]})))
+
+(rf/reg-event-fx :wallet/send-transaction
+ (fn [{:keys [db]} [sha3-pwd]]
+   (let [route (get-in db [:wallet :ui :send :route])
+         from-address (get-in db [:wallet :current-viewing-account-address])
+         to-address (get-in db [:wallet :ui :send :to-address])
+         from (:From route)
+         token (get-in db [:wallet :ui :send :token])
+         token-id (:symbol token)
+         from-asset token-id
+         to-asset token-id
+         bridge-name (:BridgeName route)
+         chain-id (:chainId from)
+         multi-transaction-command {:fromAddress from-address
+                                    :toAddress   to-address
+                                    :fromAsset   from-asset
+                                    :toAsset     to-asset
+                                    :fromAmount  (:AmountOut route)
+                                    :type        0}
+         transaction-bridge
+         [{:BridgeName bridge-name
+           :ChainID    chain-id
+           :TransferTx {:From                 from-address
+                        :To                   to-address
+                        :Gas                  (money/to-hex (:GasAmount route))
+                        :GasPrice             (money/to-hex (money/->wei :gwei
+                                                                         (:gasPrice (:GasFees route))))
+                        :Value                (:AmountOut route)
+                        :Nonce                nil
+                        :MaxFeePerGas         (money/to-hex
+                                               (money/->wei :gwei
+                                                            (:maxFeePerGasMedium (:GasFees route))))
+                        :MaxPriorityFeePerGas (money/to-hex (money/->wei :gwei
+                                                                         (:maxPriorityFeePerGas
+                                                                          (:GasFees
+                                                                           route))))
+                        :Input                ""
+                        :Data                 "0x"}}]
```

Comment:
I will find the explanation and send it on. For now this is okay to leave as is though.

Reply:
Data is a secondary feature, it is something which can be useful when making something other than an ETH transaction. It is the encoding of the contract call. We will eventually follow up to include this feature.
cc @alaibe in case I left any details